src/containers/BooksForm.js

```diff
@@ -8,11 +11,24 @@ margin-right: 3%;
 `;
 
 const SELECT = styled.select`
-padding: 5px;
+padding: 10px;
 background: brown;
 color:white;
+border: 1px solid gray;
+border-radius: 12px;
 margin-right: 2%;
+outline: none;
 `;
```

Comment:
[**Optional**]
Wow, this is great.
I think it would be nice and professional to separate your `CSS` from the logic files, right?

Reply:
Thanks @bellom